README.md

```diff
@@ -1,78 +1,174 @@
+you are working with ddev, you can run them with `ddev composer <scriptname>`.
+(You do not need to start or build the containers for this as this happens
+automatically.)
+
+The code-quality-related Composer scripts make use of the PHIVE-installed tools.
+This means that for non-ddev-based development, you need to run `phive install`
+before you can use the Composer scripts.
+
+### Available Composer scripts
+
+You can run `composer` (or `ddev composer`) to display a list of all available
+Composer commands and scripts.
+
+| Script name            | Action                        | Tool                                                                            |
+|------------------------|-------------------------------|---------------------------------------------------------------------------------|
+|`ci`                    | run code checks               | the tools listed for the individual checks                                      |
+|`ci:composer:normalize` | normalize `composer.json`     | [composer-normalize](https://github.com/ergebnis/composer-normalize)            |
+|`ci:json:lint`          | lint the JSON files           | [JSON Lint](https://github.com/Seldaek/jsonlint)                                |
+|`ci:php`                | run all static checks for PHP | the tools listed for the individual checks                                      |
+|`ci:php:copypaste`      | check for copy'n'pasted code  | [PHP Copy/Paste Detector (PHPCPD)](https://github.com/sebastianbergmann/phpcpd) |
+|`ci:php:cs-fixer`       | check the code style          | [PHP Coding Standards Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer)      |
+|`ci:php:lint`           | lint the PHP files            | `php -l`                                                                        |
+|`ci:php:sniff`          | check the code style          | [PHP_CodeSniffer (PHPCS)](https://github.com/squizlabs/PHP_CodeSniffer)         |
+|`ci:php:stan`           | check the types               | [PHPStan](https://github.com/phpstan/phpstan)                                   |
+|`ci:tests:functional`   | run the functional tests      | [PHPUnit](https://github.com/sebastianbergmann/phpunit/)                        |
+|`ci:tests:tests`        | run all PHPUnit tests         | [PHPUnit](https://github.com/sebastianbergmann/phpunit/)                        |
+|`ci:tests:unit`         | run the unit tests            | [PHPUnit](https://github.com/sebastianbergmann/phpunit/)                        |
+|`ci:ts:lint`            | lint the TypoScript files     | [TypoScript Lint:](https://github.com/martin-helmich/typo3-typoscript-lint)     |
+|`ci:yaml:lint`          | lint the YAML files           | [yaml-lint](https://github.com/j13k/yaml-lint)                                  |
+|`fix:php`               | fix the code style            | PHP Coding Standards Fixer and PHP_CodeSniffer (PHPCS)                          |
```

Comment:
maybe we should use "run all fixes for PHP" in "Action" and "the tools listed for the individual checks" instead of listing tools in "Tools"